CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 
 🔥 *Features*
 * Add `sdk.mlflow.get_current_user()` function to improve MLflow UI labeling
+* Bump Ray version to 2.5.1
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches and the blocking or non-blocking decorations to further communicate your intent.  -->
**suggestion (non-blocking):** ‏Maybe put this in the "breaking changes" to give it higher visibility.